frontend/src/components/Patchnotes.jsx

```diff
@@ -20,53 +34,68 @@ const Patchnotes = () => {
         };
 
         fetchPatchDetails();
-    }, [year, month]);
+    }, [game, year, month, version]);
 
     return (
         <div>
-            <h2>Tank Updates</h2>
-            {patchDetails.tank.length > 0 ? (
-                patchDetails.tank.map((content, index) => (
-                    <div key={index} dangerouslySetInnerHTML={{ __html: content }} />
-                ))
-            ) : (
-                <p>No tank updates found.</p>
-            )}
+            {game === 'overwatch' ? (
+                <>
+                    <h2>Tank Updates</h2>
+                    {patchDetails.tank.length > 0 ? (
+                        patchDetails.tank.map((content, index) => (
+                            <div key={index} dangerouslySetInnerHTML={{ __html: content }} />
```

Comment:
Oh I forgot to mention, there's a reason why the API is called `dangerously` set inner HTML -- let's try to avoid this if possible :)